rainier-core/src/main/scala/com/stripe/rainier/core/Discrete.scala

```diff
@@ -13,6 +13,78 @@ object Discrete {
     }
 }
 
+/**
+  * Discrete Constant (point mass) with expecation `constant`
+  *
+  * @param constant The integer value of the point mass
+  */
+final case class DiscreteConstant(constant: Real) extends Discrete {
+  val generator: Generator[Int] =
+    Generator.require(Set(constant)) { (c, n) =>
+      n.toInt(constant)
+    }
+
+  def logDensity(v: Real): Real = {
```

Comment:
style nit: we don't need braces around this method body

Reply:
definitely